tests/termsAndConditions.js

```diff
@@ -0,0 +1,61 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../src/index';
+import statusCode from '../src/helpers/constants/status.codes';
+
+const { expect } = chai;
+chai.use(chaiHttp);
+let userToken = '';
+
+describe('test the rating controller', () => {
+  it('should login a user and get a token to use against on rating article', (done) => {
+    chai
+      .request(app)
+      .post('/api/users/login')
+      .send({
+        email: 'diane@gmail.com',
+        password: 'password23423'
+      })
+      .end((err, res) => {
+        userToken = res.body.user.token;
+        done();
```

Comment:
there is a helper to get the token in helpers/test/signup. you don't need to login

Reply:
I can't use it because it is not signing up the admin and the only user allowed  and the helper is not dealing with roles